internal/controllers/addon/phase_ensure_catalogsource_test.go

```diff
@@ -19,12 +19,12 @@ import (
 func TestReconcileCatalogSource_NotExistingYet_HappyPath(t *testing.T) {
 	c := testutil.NewClient()
 	c.On("Get",
-		testutil.IsContext,
+		mock.Anything,
```

Comment:
The first argument to `Get` will still always be of type Context, right? (but just with a logger value). Why are we replacing it with `mock.Anything`?

Reply:
context.Context is just an interface and the assertion library is testing against pointer types, so while `context.TODO()` - which we are using everywhere is an `*emptyCtx`, by adding a value, we are suddenly passing an `*valueCtx` or `*cancelCtx` so all tests fail...

In the end it does not matter for our unittests, what kind of context we are using and that we are using something fitting the context.Context interface is handled by Go itself :)